Weather/emojiWeather.30m.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/local/bin/node
```

Comment:
Add jshint configuraiton here to say `esversion: 6`

Reply:
Just committed the new line. Do I need to issue a new pull request or is it set? Thanks for your help!

Reply:
Nope, should be automagicly picked up here :)